test/library/standard/Socket/SKIPIF

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+
+has_libevent = subprocess.getoutput("ldconfig -v | grep libevent - | wc -l")
+has_libevent = has_libevent.split("\n")
+print(os.getenv('CHPL_TASKS') != 'fifo' and os.getenv('CHPL_TARGET_COMPILER') != 'llvm' and int(has_libevent[-1]) != 0)
```

Comment:
We need to skip if CHPL_TASKS != qthreads, right? This script needs to return True if the tests should be skipped and it doesn't look like it's doing that. Maybe it is reversed.

Reply:
Yep made changes